### PR TITLE
Add links to the notebook for the tutorial

### PR DESCRIPTION
**Pull request by eric-wieser**
https://galgebra--417.org.readthedocs.build/en/417/tutorials/algebra.html

### CODE REVIEW

**Comment by utensil** [comment]
Everything is ok with this PR, but for https://nbviewer.jupyter.org/urls/galgebra--417.org.readthedocs.build/en/417/tutorials/algebra.ipynb :

image

File: doc/conf.py [hunk 1]
```diff
@@ -68,6 +68,55 @@
...
+            Interactive online version:
+            <a href="https://mybinder.org/v2/gh/pygae/galgebra/{{ git_ref|e }}?filepath={{ docname|e }}"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg" style="vertical-align:text-bottom"></a>.
+        -->
+      </p>
+      <script>
+        if (document.location.host) {
+          var p = document.currentScript.previousSibling.previousSibling;
+          var a = document.createElement('a');
+          a.innerHTML = 'View in <em>nbviewer</em>';
+          a.href = `https://nbviewer.jupyter.org/url${
```
Comment (utensil) [inline doc/conf.py:91]: I definitely prefer a github based nbviewer url. Any particular reason/difficulty made you choose the URL based approach?

Comment (eric-wieser) [reply doc/conf.py:91]: Mainly because it meant I could copy-paste the approach from clifford.

**Comment by eric-wieser** [comment]
> Everything is ok with this PR, but for

Patch submitted upstream as https://github.com/jupyter/nbconvert/pull/1278

File: doc/conf.py (hunk 1 above)
Comment (eric-wieser) [reply doc/conf.py:91]: It renders a lot faster too.

**Pull request merged**